deps/pylib.sh

```diff
@@ -1 +1,2 @@
 pip install requests_toolbelt filemagic ftputil
+pip install git+https://github.com/metabolite-atlas/metatlas
```

Comment:
Ok, now pip installation finishes successfully. But "metatlas" folder doesn't appear in python site-packages. There is only "metatlas-0.2-py2.7.egg-info" folder with dependencies (and other text files). So in particular there is no "mzml_loader.py" anywhere. I tried it in two different architectures (linux and mac). Did you try to make this command on clean python environment?